Platform/ElkhartlakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c

```diff
@@ -239,6 +241,37 @@ TccModePreMemConfig (
   return Status;
 }
 
+/**
+  Update S0ix flag
+**/
+VOID
+UpdateS0ixStatus (
+  VOID
+)
+{
+  FEATURES_CFG_DATA             *FeaturesCfgData;
+  BOOLEAN                       PchSciSupported;
+
+  FeaturesCfgData = (FEATURES_CFG_DATA *) FindConfigDataByTag (CDATA_FEATURES_TAG);
+  if (FeaturesCfgData == NULL) {
+    DEBUG ((DEBUG_INFO, "Failed to find Cfg Config Data! S0ix setting failed.\n"));
+    return;
+  }
+  PchSciSupported               = PchIsSciSupported ();
+  /**
+  EHL S0ix condition depends on:
+  - user turn on via config data
+  - not FUSA sku
+  - TCC config turns it on
+  **/

```

Comment:
EDKII use "//" instead of "/*"  "*/" for comments inside a function. nice to update it

Reply:
hahaha old habit from coreboot. nice catch :)

Reply:
Done.